src/components/Calendar/CalendarPopup.jsx

```diff
@@ -0,0 +1,181 @@
+    setSelectedTime(e.target.value);
+  };
+
+  const handleSave = async (e) => {
+    const mediaType = isAppointment ? "Dein Termin" : "Dein ToDo";
+    if (!title || !selectedGroup || !selectedTime || !selectedDate) {
+      setSnackbarContent({
+        message: "Bitte fülle alle Felder aus",
+        status: "error",
+        open: true,
+      });
+      return;
+    }
+    await addCalendar({
+      date: selectedDate,
+      time: selectedTime,
+      groupID: selectedGroup.groupID,
+      color: selectedGroup.color,
+      title: title,
+      isAppointment: isAppointment,
+    });
+    close();
+    setSelectedGroup("");
+    setTitel("");
+    setSelectedDate("");
+    setSelectedTime("");
+    setSnackbarContent({
+      message: `${mediaType} wurde erfolgreich hinzugefügt`,
+      status: "success",
+      open: true,
+    });
+  };
+
+  return (
+    <>
+      <Snackbar snackbarContent={snackbarContent} setSnackbarContent={setSnackbarContent} />
+      <Dialog open={open} onClose={close}>
+        <DialogTitle id="termine-todos-hinzufügen" className={classes.dialogTitle}>
+          Termin/ToDos hinzufügen
+        </DialogTitle>
+        <DialogContent dividers className={classes.content}>
+          <form className={classes.form} noValidate autoComplete="off">
+            <TextField id="title" label="Titel" onChange={handleTitel} value={title} />
+            <FormControl>
+              <InputLabel htmlFor="selectGroup">Gruppe</InputLabel>
+              <Select
+                native
+                value={`${selectedGroup.groupID}/${selectedGroup.color}`}
+                onChange={handleDropDown}
+              >
+                <option aria-label="None" value="" />
+                {!isLoading &&
+                  Object.entries(userData.groups).map((group, idx) => (
+                    <option key={idx} value={`${group[0]}/${group[1].color}`}>
+                      {group[1].name}
+                    </option>
+                  ))}
+              </Select>
+            </FormControl>
+            <TextField
+              id="date"
+              onChange={handleSelectDate}
+              label="Tag"
+              type="date"
+              value={selectedDate}
+              className={classes.textField}
+              InputLabelProps={{
+                shrink: true,
+              }}
+            />
+            <TextField
+              id="time"
+              onChange={handleSelectTime}
+              label="Zeit"
+              type="time"
+              value={selectedTime}
+              className={classes.textField}
+              InputLabelProps={{
+                shrink: true,
+              }}
+            />
+            <FormControl>
+              <InputLabel htmlFor="selectGroup">Eintragstyp</InputLabel>
+              <Select native value={isAppointment} onChange={handleAppointment}>
+                <option value={true}>Termin</option>
+                <option value={false}>ToDos</option>
```

Comment:
```suggestion
                <option value={false}>ToDo</option>
```
ToDo mach in meinen Augen mehr sind weil man ja nur eines erstellt